### ISSUE TITLE
Allow raw axios headers

### ISSUE THREAD

**Issue by sharifzadesina** [tronprotocol/tronweb#521]
On v6, This code has type error:

```js
import { TronWeb } from "tronweb";

var tw = new TronWeb({
  fullHost: '',
  headers: { 'test': 'test' },
});
```

But this code is fine:

```js
const instance = axios.create({
  headers: { 'test': 'test' }
});
```

The reason is you are using `AxiosRequestHeaders` instead of `RawAxiosRequestHeaders | AxiosHeaders`.
`AxiosRequestHeaders`  is equal to `RawAxiosRequestHeaders & AxiosHeaders` which means having feathers of both `RawAxiosRequestHeaders` and `AxiosHeaders`.

**Comment by start940315** [comment]
We are consider to use our inner type HeadersType in file `src/lib/providers/HttpProvider.ts`. Thanks!

**Comment by start940315** [comment]
We just published `TronWeb@v6.0.0-beta.4` which fixed this problem. Welcome to try:
```bash
npm install tronweb@beta
```
